src/auction.js

```diff
@@ -515,84 +539,54 @@ export function getStandardBidderSettings(mediaType) {
-        key: CONSTANTS.TARGETING_KEYS.CPM,
-        val: function (bidResponse) {
-          return bidResponse.cpm;
-        }
-      }, {
-        key: CONSTANTS.TARGETING_KEYS.PRICE_BUCKET,
-        val: function (bidResponse) {
-          if (granularity === CONSTANTS.GRANULARITY_OPTIONS.AUTO) {
-            return bidResponse.pbAg;
-          } else if (granularity === CONSTANTS.GRANULARITY_OPTIONS.DENSE) {
-            return bidResponse.pbDg;
-          } else if (granularity === CONSTANTS.GRANULARITY_OPTIONS.LOW) {
-            return bidResponse.pbLg;
-          } else if (granularity === CONSTANTS.GRANULARITY_OPTIONS.MEDIUM) {
-            return bidResponse.pbMg;
-          } else if (granularity === CONSTANTS.GRANULARITY_OPTIONS.HIGH) {
-            return bidResponse.pbHg;
-          } else if (granularity === CONSTANTS.GRANULARITY_OPTIONS.CUSTOM) {
-            return bidResponse.pbCg;
-          }
-        }
-      }, {
-        key: CONSTANTS.TARGETING_KEYS.SIZE,
-        val: function (bidResponse) {
-          return bidResponse.size;
-        }
-      }, {
-        key: CONSTANTS.TARGETING_KEYS.DEAL,
-        val: function (bidResponse) {
-          return bidResponse.dealId;
-        }
-      },
-      {
-        key: CONSTANTS.TARGETING_KEYS.SOURCE,
-        val: function (bidResponse) {
-          return bidResponse.source;
+      createKeyVal(TARGETING_KEYS.BIDDER, 'bidderCode'),
+      createKeyVal(TARGETING_KEYS.AD_ID, 'adId'),
+      createKeyVal(TARGETING_KEYS.PRICE_BUCKET, function(bidResponse) {
+        if (granularity === CONSTANTS.GRANULARITY_OPTIONS.AUTO) {
+          return bidResponse.pbAg;
+        } else if (granularity === CONSTANTS.GRANULARITY_OPTIONS.DENSE) {
+          return bidResponse.pbDg;
+        } else if (granularity === CONSTANTS.GRANULARITY_OPTIONS.LOW) {
+          return bidResponse.pbLg;
+        } else if (granularity === CONSTANTS.GRANULARITY_OPTIONS.MEDIUM) {
+          return bidResponse.pbMg;
+        } else if (granularity === CONSTANTS.GRANULARITY_OPTIONS.HIGH) {
+          return bidResponse.pbHg;
+        } else if (granularity === CONSTANTS.GRANULARITY_OPTIONS.CUSTOM) {
+          return bidResponse.pbCg;
         }
-      },
-      {
-        key: CONSTANTS.TARGETING_KEYS.FORMAT,
-        val: function (bidResponse) {
-          return bidResponse.mediaType;
-        }
-      },
-      {
-        key: CONSTANTS.TARGETING_KEYS.TIME_TO_RESPOND,
-        val: function (bidResponse) {
-          return bidResponse.timeToRespond;
-        }
-      },
-      {
-        key: CONSTANTS.TARGETING_KEYS.AUCTION_ID,
-        val: function (bidResponse) {
-          return bidResponse.auctionId;
-        }
-      },
+      }),
+      createKeyVal(TARGETING_KEYS.SIZE, 'size'),
+      createKeyVal(TARGETING_KEYS.DEAL, 'dealId'),
+      createKeyVal(TARGETING_KEYS.SOURCE, 'source'),
+      createKeyVal(TARGETING_KEYS.FORMAT, 'mediaType'),
+
+      createKeyVal(TARGETING_KEYS.CPM, 'cpm'),
```

Comment:
These 3 lines were the only ones that were actually edited. Changed them they use the same way to create KeyVals as the rest of the function.
Everything else comes from the upgrade.